uaclient/cli/enable.py

```diff
@@ -11,25 +13,138 @@
     status,
     util,
 )
+from uaclient.api.u.pro.services.dependencies.v1 import _dependencies
+from uaclient.api.u.pro.status.enabled_services.v1 import _enabled_services
 from uaclient.cli import cli_util, constants
 from uaclient.entitlements.entitlement_status import (
     CanEnableFailure,
     CanEnableFailureReason,
 )
 
-event = event_logger.get_event_logger()
 LOG = logging.getLogger(util.replace_top_level_logger_name(__name__))
 
 
+class CLIEnableProgress(api.AbstractProgress):
+    def __init__(self, messaging):
+        self.messaging = messaging
```

Comment:
Any reason why we must create the object if the messages from the entitlement ? I am wondering if it is simpler to always pass the event and payload to `emit`

Reply:
Yep that's a good idea, I'll refactor to do that